processes/refresh.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+while IFS= read -r -d '' file
```

Comment:
why not just 

```
for file in $(find . -name "*.dot" -print0)
do
...
done
```

Reply:
You can ask shellcheck why - it made me do it 😄

Reply:
hmm might be whitespace safe 🤔

Reply:
Yeah apparently this is for whitespace safety, TIL https://unix.stackexchange.com/questions/174016/how-do-i-use-null-bytes-in-bash